kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/FragmentActivities.kt

```diff
@@ -0,0 +1,108 @@
+package com.taskworld.kxandroid.support.v4
+
+import android.content.Context
+import android.graphics.Matrix
+import android.graphics.RectF
+import android.os.Parcelable
+import android.support.v4.app.FragmentActivity
+import android.support.v4.app.SharedElementCallback
+import android.view.View
+
+/**
+ * Created by tipatai on 4/26/16 AD.
+ */
+
+class KxSharedElementCallback {
```

Comment:
I think you'd better going something along this line. So, it is consistent.

https://github.com/taskworld/KxAndroid/blob/master/kxandroid/src/main/java/com/taskworld/kxandroid/TextViews.kt#L37